coreneuron/permute/cellorder.cu

```diff
@@ -0,0 +1,101 @@
+#ifdef ENABLE_CUDA_INTERFACE
+
+//#include <iostream>
+//#include <stdio.h>
+
+#include "coreneuron/permute/cellorder.hpp"
+#include "coreneuron/network/tnode.hpp"
+#include "coreneuron/sim/multicore.hpp"
+
+namespace coreneuron {
+
+#define GPU_A(i)      nt->_actual_a[i]
+#define GPU_B(i)      nt->_actual_b[i]
+#define GPU_D(i)      nt->_actual_d[i]
+#define GPU_RHS(i)    nt->_actual_rhs[i]
+#define GPU_PARENT(i) nt->_v_parent_index[i]
```

Comment:
nitpick: this is older pattern; as we want to write code with less macros, would be nice to avoid this?